index.js

```diff
@@ -21,12 +22,15 @@ Module._nodeModulePaths = function (from) {
 
 var oldResolveFilename = Module._resolveFilename
 Module._resolveFilename = function (request, parent, isMain) {
-  for (var alias in moduleAliases) {
+  for (var i = moduleAliasNames.length; i-- > 0;) {
+    var alias = moduleAliasNames[i]
     if (isPathMatchesAlias(request, alias)) {
       request = nodePath.join(
         moduleAliases[alias],
         request.substr(alias.length)
       )
+      // Only use the first match
+      break
```

Comment:
Overwriting our previous matches results in unexpected behaviour for future matches due to the `String.prototype.substr()` call on line 30. Altering this did not break any tests and results in a definitive match on the longest alias.